package.json

```diff
@@ -24,6 +24,14 @@
   ],
   "repository": "git@github.com:Automattic/wpcom.js.git",
   "license": "MIT",
+  "babel": {
+    "presets": [ "es2015", "stage-2" ]
+  },
+  "files": [
```

Comment:
it allows run tests written in ES6 with mocha on the fly. I've added the parameter `--compilers js:babel-register` as well.